components/eam/src/physics/crm/physpkg.F90

```diff
@@ -1805,1229 +1180,654 @@ subroutine tphysac (ztodt,   cam_in,  &
-       snow_pcw(:ncol) = snow_pcw_macmic(:ncol)/cld_macmic_num_steps
-       prec_str(:ncol) = prec_pcw(:ncol) + prec_sed(:ncol)
-       snow_str(:ncol) = snow_pcw(:ncol) + snow_sed(:ncol)
-
-     end if !microp_scheme
-
-   !================================================================================================
-   !------------------------------------------------------------------------------------------------
-   ! CRM Physics
-   !------------------------------------------------------------------------------------------------
-   !================================================================================================
-   if (use_MMF) then
-      crm_run_time = ztodt
-      !-------------------------------------------------------------------------
-      ! Recall the state after dynamics
-      !-------------------------------------------------------------------------
-      call crm_recall_state_tend(state, tend, pbuf)
-
-      !-------------------------------------------------------------------------
-      ! Apply surface fluxes if using MMF_FLUX_BYPASS
-      !-------------------------------------------------------------------------
 #if defined( MMF_FLUX_BYPASS )
-      call crm_surface_flux_bypass_tend(state, cam_in, ptend)
-      call physics_update(state, ptend, ztodt, tend)  
-      call check_energy_chng(state, tend, "crm_tend", nstep, crm_run_time,  &
-                             cam_in%shf(:), zero, zero, cam_in%cflx(:,1)) 
+  call crm_surface_flux_bypass_tend(state, cam_in, ptend)
+  call physics_update(state, ptend, ztodt, tend)  
+  call check_energy_chng(state, tend, "crm_tend", nstep, crm_run_time,  &
+                         cam_in%shf(:), zero, zero, cam_in%cflx(:,1)) 
 #endif
-      !-------------------------------------------------------------------------
-      ! Initialize variabale for ECPP data
-      !-------------------------------------------------------------------------
+
+  !-----------------------------------------------------------------------------
+  ! Initialize variabale for ECPP data
+  !-----------------------------------------------------------------------------
 #if defined( ECPP )
-      if (use_ECPP) then
-         call crm_ecpp_output%initialize(pcols,pver)
-      end if ! use_ECPP
+  if (use_ECPP) call crm_ecpp_output%initialize(pcols,pver)
 #endif
-      !-------------------------------------------------------------------------
-      ! Run the CRM 
-      !-------------------------------------------------------------------------
-      call crm_physics_tend(ztodt, state, tend,ptend, pbuf, cam_in, cam_out,    &
-                            species_class, crm_ecpp_output, mmf_clear_rh,       &
-                            mmf_qchk_prec_dp, mmf_qchk_snow_dp, mmf_rad_flux)
+  !-----------------------------------------------------------------------------
+  ! Run the CRM 
+  !-----------------------------------------------------------------------------
+  crm_run_time = ztodt
```

Comment:
What's the point of defining a second variable to represent the timestep?

Reply:
yea, that's correct. I'll remove it.

Reply:
done.